### ISSUE TITLE
please support to post json body just use string content

### ISSUE THREAD

**Issue by SeanCai** [rybalkinsd/kohttp#51]

**Comment by shtykh** [comment]
In the next release https://github.com/rybalkinsd/kohttp/tree/release/0.7.0#post-with-json-body

**Comment by rybalkinsd** [comment]
Feature implemented in 0.7.0 release
In your case:
```kotlin
body(type = "application/json") {
    string("your json content")
}
```
Also checkout docs for more details

@SeanCai
